testsuite/pytests/sli2py_stimulating/test_ou_noise_generator.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file should be directly in `pytests`. The `sli2py_*` directories are only for tests ported from SLI.